src/pyhf/infer/calculators.py

```diff
@@ -524,7 +524,7 @@ def expected_pvalues(self, sig_plus_bkg_distribution, bkg_only_distribution):
             :math:`\mathrm{CL}_{b}`, and :math:`\mathrm{CL}_{s}`.
         """
         # Calling pvalues is easier then repeating the CLs calculation here
-        tb, _ = get_backend()
+        _, _ = get_backend()
```

Comment:
If we're unpacking `get_backend` into empty variables then it doesn't have any effect, right? So I think we can remove this? Or am I forgetting something about `get_backend` side effects? This `get_backend()` call was introduced back in PR https://github.com/scikit-hep/pyhf/pull/790.

```suggestion
```